src/redux/books/Books.js

```diff
@@ -0,0 +1,36 @@
+import { v4 as uuidv4 } from 'uuid';
+
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] (OPTIONAL) This is not really optional but as the first reviewer did not point it out, I'll leave it as optional. Kindly make sure that the values of your action type follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux). Therefore it should be something like this: `const ADD_BOOK = bookstore/books/ADD_BOOK`. Kindly rectify this 💯